python/tutorials/09-persistent-matmul.py

```diff
@@ -604,7 +631,8 @@ def validate(M, N, K, dtype):
     validate(32, 32, 32, dtype)
     validate(8192, 8192, 512, dtype)
 
-    proton.start("matmul", hook="triton")
-    for K in range(args.K_range[0], args.K_range[1] + 1, args.K_step):
-        bench(K, dtype)
-    proton.finalize()
+    if 1:
```

Comment:
can we make this run based on a command line?

Reply:
I don't think this is even necessary, should be OK to run this always?